lib/widgets/show_functions.dart

```diff
@@ -245,4 +245,52 @@ class ShowFunctions extends NFShowFunctions {
       ui: ui,
     );
   }
+
+  Future<void> showRadio<T>({
+    required BuildContext context,
+    required String title,
+    required List<T> items,
+    required String Function(T) itemTitleBuilder,
+    required ValueSetter<T> onItemSelected,
+    required ValueGetter<T> groupValueGetter,
+  }) {
+    final l10n = getl10n(context);
+    final theme = Theme.of(context);
+
+    Widget buildItem(T item) {
+      return Theme(
+        data: Theme.of(context).copyWith(
+          splashFactory: NFListTileInkRipple.splashFactory,
+        ),
+        child: Builder(
+          // i need the proper context to pop the dialog
+          builder: (context) => AppRadioListTile<T>(
+            title: Text(
+              itemTitleBuilder(item),
+              style: theme.textTheme.subtitle1,
+            ),
+            value: item,
+            groupValue: groupValueGetter(),
+            onChanged: (value) {
+              onItemSelected(value);
+              Navigator.pop(context);
+            },
+          ),
+        ),
+      );
+    }
+
+    return ShowFunctions.instance.showAlert(
+      context,
+      ui: theme.systemUiThemeExtension.modalOverGrey,
+      title: Text(l10n.sort),
```

Comment:
```suggestion
      title: Text(title),
```
I noticed this in the video.